index.js

```diff
@@ -449,6 +471,7 @@ function TorrentGraph (root) {
     choke: choke,
     seed: seed,
     rate: rate,
+    on: on,
```

Comment:
This is fine for now, but we should probably change this to return an `EventEmitter` so that other methods like `removeListener`, etc. work as expected.